src/model_config_tests/qa/test_access_esm1p5_config.py

```diff
@@ -24,9 +24,9 @@
 VALID_KEYWORDS: set[str] = {"global", "access-esm1.5"}
 VALID_NOMINAL_RESOLUTION: str = "100 km"
 VALID_REFERENCE: str = "https://doi.org/10.1071/ES19035"
-VALID_PREINDUSTRIAL_START: dict[str, int] = {"year": 101, "month": 1, "day": 1}
-VALID_HISTORICAL_START: dict[str, int] = {"year": 1850, "month": 1, "day": 1}
-VALID_RUNTIME: dict[str, int] = {"year": 1, "month": 0, "day": 0}
+VALID_PREINDUSTRIAL_START: dict[str, int] = {"year": 101, "month": 1, "days": 1}
+VALID_HISTORICAL_START: dict[str, int] = {"year": 1850, "month": 1, "days": 1}
+VALID_RUNTIME: dict[str, int] = {"years": 1, "months": 0, "days": 0}
```

Comment:
Do these fields need to be plural only OR plural & singular?

Reply:
Plural only for `VALID_RUNTIME`. The updated way is the way that Payu understands it. It's not internally consistent, but that is a pull request for Payu first.